sycl/doc/developer/ABIPolicyGuide.md

```diff
@@ -87,31 +87,42 @@ There is a set of tests to help identifying ABI changes:
   Please, prefer updating the test files with the above command. The checker
   script automatically sorts symbols. This would allow developers to avoid
   large diffs and help maintainers identify the nature of ABI changes.
-* `test/abi/layout*` and `test/abi/symbol_size_alignment.cpp` are a group of 
+* `test/abi/layout*` and `test/abi/symbol_size_alignment.cpp` are a group of
   tests to check the internal layout of some classes. The `layout*` tests check 
-  some of API classes for layout changes, while `symbol_size_alignment` only 
+  some of API classes for layout changes, while `symbol_size_alignment` only
   checks `sizeof` and `alignof` for API classes. Changing the class layout is a 
   breaking change.
 
 ## Changing ABI
 
-**Note (October, 2020)**: DPC++ runtime and compiler ABI is currently in frozen
-state. This means that no ABI-breaking changes will be accepted by default.
-Project maintainers may still approve breaking changes in some cases. Please,
-try to avoid any breaking changes until freeze is lifted. If you need to change
-existing functionality, consider adding new APIs instead of replacing them.
-Also, please, avoid any changes, mentioned in the [Intro](#intro) section as
-breaking. Refer to the above guide to distinguish breaking and non-breaking
-changes. If not sure, do not hesitate to ask code owners for help.
+Generally DPC++ runtime and compiler ABI is frozen and ABI-breaking changes are
+not accepted by default since Oct 2020. Please, try to avoid making any breaking
```

Comment:
```suggestion
not accepted by default since Oct 2020. Please try to avoid making any breaking
```
Not completely sure I am right here but the comma looks off. 😄